python/cudf_polars/cudf_polars/utils/config.py

```diff
@@ -77,9 +77,11 @@ class ParquetOptions:
     Parameters
     ----------
     chunked
-        Whether to use libcudf's ``ChunkedParquetReader`` to read the parquet
-        dataset in chunks. This is useful when reading very large parquet
-        files.
+        Whether to use libcudf's ``ChunkedParquetReader`` or ``ChunkedParquetWriter``
+        to read/write the parquet dataset in chunks. This is useful when reading/writing
+        very large parquet files.
+    n_output_chunks
```

Comment:
How should I think about this parameter with the multi-partition streaming executor? IIUC, it's not currently honored, and we write out a single output chunk per input chunk.

Should we make it clear that this only applies to the in-memory executor? (And do we have any other ParquetOptions that are executor-specific, or have we done a good job keeping executor-specific options on the executors?)

I don't think we should try to honor this by repartitioning to a single partition before doing the sink. I'm less certain about whether we should reframe this parameter as `n_output_chunks_per_partition`, but I think I still lean towards not doing that.

Reply:
xref https://github.com/rapidsai/cudf/pull/19317